### ISSUE TITLE
Updates don't update if Notepad++ is open 

### ISSUE THREAD

**Issue by cbiere** [notepad-plus-plus/notepad-plus-plus#14198]
### Description of the Issue

When running the Installer EXE while Notepad++ is running and in use, the Update does seemingly succeed i.e. appwiz.cpl shows the new version corresponding to the installer. When you restart Notepad++, the About dialog (F1) still shows the old version. The installation folder also shows that the dates of most files have not changed especially notepad++.exe.
### Steps to Reproduce the Issue

1. Run an older version of Notepad++.
2. Run the installer executable for a newer version.
3. Witness that the installer succeeds but the old version is still installed.

### Expected Behavior

The update should fail or resume when Notepad++ is closed.

### Actual Behavior

Some files have been updated. Probably those that were not in use   but many files especially the main executable have not been updated.

This is not a new issue and happens on Windows 10 as well as Windows 11. So the specific version doesn't matter. I'm always using the 64-bit version, if that matters.

### Workaround 

As a workaround, the user has to make sure all Notepad++ processes are closed before starting the update or uninstall the current version before installing the new one. When you're doing this manually it's not a big issue but it matters a lot for automatic deployment. Basically you cannot trust the package information of Windows and you might be using an old version eith vulnerabilities while assuming the latest version is actually installed.

**Comment by CennoxX** [comment]
This is fixed by #14251

**Comment by donho** [comment]
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/83f1195bc1f8d84080a4a243019f155a2f45f430
